Editor/Resources/EditorWindow/EditorMenu.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using AmazonGameLift.Editor;
+using Editor.Window;
+using UnityEditor;
+using UnityEngine;
+
+namespace Editor.Resources.EditorWindow
+{
+    internal static class EditorMenu
+    {
+        private static GameLiftPlugin GetPluginWindow()
+        {
+            var inspectorType = Type.GetType("UnityEditor.GameView,UnityEditor.dll");
```

Comment:
Should we do this once and store as a static variable? I'm not sure of the performance implications, but it looks like we're loading a dll each time

Reply:
Set as static variable